src/pages/index.js

```diff
@@ -111,6 +184,50 @@ profileEditButton.addEventListener("click", () => {
 });
 addNewCardButton.addEventListener("click", () => cardPopupForm.open());
 
+/*=============================================
+=            popup confirmation            =
+=============================================*/
+const deleteConfirmPopup = new PopupWithConfirmation("#confirm-modal");
+deleteConfirmPopup.setEventListeners();
+
+function handleDeleteClick(cardElement) {
+  deleteConfirmPopup.open();
+
+  deleteConfirmPopup.setSubmitAction(() => {
+    deleteConfirmPopup.setLoading(true);
+    api
+      .fetchDeleteCard(cardElement.getId())
+      .then(() => {
+        cardElement.removeCard();
+        deleteConfirmPopup.close();
+      })
+      .catch(console.error)
+      .finally(() => deleteConfirmPopup.setLoading(false));
+  });
+}
+
+/*=============================================
+=            like and dislike            =
+=============================================*/
+
+function handleLikeClick(cardElement) {
+  if (cardElement.getLikes()) {
+    api
+      .fetchDisLikeCard(cardElement.getId())
+      .then((res) => {
+        cardElement.renderLikes(res.isliked);
+      })
+      .catch(console.error);
+  } else {
+    api
+      .fetchLikeCard(cardElement.getId())
+      .then((res) => {
+        cardElement.renderLikes(res.isliked);
+      })
+      .catch(console.error);
+  }
+}
+
 /*=============================================
 =            validators            =
```

Comment:
* The avatar form is not validated


Please, create a validator for the avatar form